plugin.video.montreal.greek-tv/resources/language/resource.language.el_gr/strings.po

```diff
@@ -0,0 +1,40 @@
+# Addon Name: Montreal Greek TV
+# Addon id: plugin.video.montreal.greek-tv
+# Addon Provider: Twilight0
+msgid ""
+msgstr ""
+"Project-Id-Version: Montreal Greek TV\n"
+"POT-Creation-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"PO-Revision-Date: 2016-06-22 11:29+0000\n"
+"Last-Translator: twilight0\n"
+"Language-Team: English (http://www.domain.com)\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Language: el\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+msgctxt "#30001"
+msgid "General"
+msgstr "Γενικά"
+
+msgctxt "#30002"
+msgid "Use old radio stream"
+msgstr "Κάνε χρήση της παλαιάς ροής ραδιοφώνου"
+
+msgctxt "#30003"

```

Comment:
any reason for those empty strings?

Reply:
Absolutely no reason for their existence. I have an addon as a template with empty strings on stand by, hense not deleting them, because I forgot.